### ISSUE TITLE
Calendar color update

### ISSUE THREAD

**Issue by handuozh** [akhater/Lupin#8]
Hi. 
Firstly really love this tool and it is now my daily tool!

I am wondering how does Lupin scan the daily notes and update the state in a repeated way?

E.g. the date of generating the calendar was 19th and today is already 23th but the bold date is still 19.

![image](https://user-images.githubusercontent.com/40494279/108797917-65b16b80-75c7-11eb-83e8-78fb2f743bfa.png)

Also wish you could scan the files and if it is empty, also label that daily note as "void", the same as the dates with no files.

**Comment by akhater** [comment]
Hi @handuozh 
Lupin scans at start up and then every GitHubUpdateFrequency minutes, by default it is every 720 min / 12 hrs

it is really weird that it was no updated for you in 3 days, in your terminal are you seeing a "Scheduled Housekeeping" message appearing? it is when it is building them 

for the second request sure I can do that, I was./am reluctant since it will take more time but no worries I'll do it

**Comment by handuozh** [comment]
Sorry for the late reply. I just checked the output and really found some errors:
```
Scheduled Housekeeping ...

No error handlers are registered, logging exception.

Traceback (most recent call last):

  File "/usr/local/lib/python3.6/site-packages/apscheduler/executors/base.py", line 125, in run_job

    retval = job.func(*job.args, **job.kwargs)

  File "main.py", line 281, in scheduledHousekeeping

    Git2Json()

  File "/home/christlurker/softwares/Lupin.LogSeqPyBot/git.py", line 156, in Git2Json

    AllFilesContent.append(getGitFileContent(content))

  File "/home/christlurker/softwares/Lupin.LogSeqPyBot/git.py", line 123, in getGitFileContent

    return file.decoded_content.decode("utf-8")  # Get raw string data

UnicodeDecodeError: 'utf-8' codec can't decode byte 0x89 in position 0: invalid start byte
```

**Comment by handuozh** [comment]
For the 2nd request, it is not that important, actually. 😆  No need to worry about this.

**Comment by akhater** [comment]
> ```
> UnicodeDecodeError: 'utf-8' codec can't decode byte 0x89 in position 0: invalid start byte
> ```

very interesting 🤔I have no clue what this error actually means but I will take a look at that

thanks for the feedback

**Comment by akhater** [comment]
is it possible that you have in your logseg/pages or /journal directories a non-text file? like image etc...

**Comment by handuozh** [comment]
I don't have non-text files but have some non-ascii files like Chinese characters and emoji ❤️ under pages/. Not sure if they could be decoded by utf8. U can check out my public repo as u like https://github.com/handuozh/logseq-notes

**Comment by akhater** [comment]
i'll clone your public repo and give it a try
please allow me some time

**Comment by akhater** [comment]
seems I was right "pages/dynaslam.png" no prob easier than I thought

**Comment by akhater** [comment]
Hotfix released in https://github.com/akhater/Lupin/commit/e90b14c4b3eadeccdb21ccb30ea9d39b144c3842

should be working now. I'd appreciate your feedback

**Comment by handuozh** [comment]
> Hotfix released in [e90b14c](https://github.com/akhater/Lupin/commit/e90b14c4b3eadeccdb21ccb30ea9d39b144c3842)
> 
> should be working now. I'd appreciate your feedback

Thank you so much and sorry for the late reply and for wasting some of your time checking my repo :+1:  

When I replied to you, I was in bed browsing the mobile Github so cannot easily filter out file types ignoring that .png file.  :rofl: 
Now it works

**Comment by akhater** [comment]
No matter how you spin it, it was an unhandled error from my side and I appreciate your help on this

I will remove the feature request tag from this one, please feel free to open another issue for that
